templates/index.html

```diff
@@ -52,6 +52,10 @@
         <style>
             .o-a {overflow-x: auto;}
             .mw-800p {max-width: 800px; margin: auto;}
+            .bg-gray-700 {background-color: #4A5568;}
+            .hide-scrollbar {-ms-overflow-style: none; scrollbar-width: none;}
+            .hide-scrollbar::-webkit-scrollbar {display: none;}
```

Comment:
On mobile, the scroll bars like on Windows are not shown. I think we should keep them.